lib/src/terminal_view.dart

```diff
@@ -422,9 +421,12 @@ class TerminalViewState extends State<TerminalView> {
 
     final handled = widget.terminal.keyInput(
       key,
-      ctrl: event.isControlPressed,
-      alt: event.isAltPressed,
-      shift: event.isShiftPressed,
+      ctrl: event.physicalKey == PhysicalKeyboardKey.controlLeft ||
```

Comment:
I think you'll find that the shift and control keys don't work as expected in your terminal editor. According to the [`RawKeyEvent` docs](https://api.flutter.dev/flutter/services/RawKeyEvent/isShiftPressed.html) the correct replacement for `isShiftPressed` is [`HardwareKeyboard.isShiftPressed`](https://api.flutter.dev/flutter/services/HardwareKeyboard/isShiftPressed.html). 

See https://github.com/TerminalStudio/xterm.dart/pull/190/files#diff-d62cc3ae7257875f8ff5223bcb0ba25023d5e12166597467e693d3f10378b87eR423 for my implementation. 

Hth!